packages/synapse-react-client/src/mocks/msw/util/BasicMockedCrudService.ts

```diff
@@ -48,7 +48,7 @@ export default class BasicMockedCrudService<
   }
 
   create(newItem: SetOptional<T, TIDField>): T {
-    const cloned = cloneDeep(newItem)
+    const cloned = cloneDeep(newItem) as unknown as T
```

Comment:
Some typescript behavior changed in `type-fest`, so add some casts to fix it in this test utility.